### PR TITLE
6 functionality to get all products

### LINKED ISSUE #6: Functionality to get all products

### PR DESCRIPTION
**Pull request by Smita-Sonu**
- [x] get all products

### CODE REVIEW

**Comment by Smita-Sonu** [comment]
get all products

File: backend/src/main/java/com/ApparelAvenue/backend/controller/ProductController.java [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/api/v1/products")
 public class ProductController {
+    private final ProductService productService;
+
+    public ProductController(ProductService productService) {
```
Comment (smruti-banty) [inline backend/src/main/java/com/ApparelAvenue/backend/controller/ProductController.java]: Remove constructor, You are using annotation to create it

**Comment by Smita-Sonu** [comment]
get all products

File: backend/src/main/java/com/ApparelAvenue/backend/service/ProductService.java [hunk 2]
```diff
@@ -24,4 +24,5 @@ public interface ProductService {
     List<Product> getProducts();
 
     Product getProductById(String id);
+
```
Comment (smruti-banty) [inline backend/src/main/java/com/ApparelAvenue/backend/service/ProductService.java]: Remove line break

**Review by smruti-banty — CHANGES_REQUESTED** [review]

**Review by smruti-banty — APPROVED** [review]
LGTM.

**Pull request merged**